src/pseudopeople/interface.py

```diff
@@ -127,7 +169,8 @@ def generate_decennial_census(
     year: Optional[int] = 2020,
     state: Optional[str] = None,
     verbose: bool = False,
-) -> pd.DataFrame:
+    engine: Literal["pandas", "modin"] = "pandas",
+) -> DATAFRAME:
```

Comment:
What is the difference between a pandas DataFrame and modin DataFrame? Is it reasonable for us to expect users to know what to do with each of them? I always have some concerns with variable return types, as it's a good opportunity for confusion.

Reply:
We discussed this in real time, but just for posterity:

- Modin dataframes aim to faithfully replicate the Pandas dataframe API, so it makes sense conceptually to return a union of these types. In most cases, the code someone runs after calling generate_X would not change at all between Modin and Pandas return type.
- This should be less confusing than e.g. pandas.squeeze return types, because the return type depends only on the `engine` argument to the function.